server/liveblog/embed/templates/embed.html

```diff
@@ -1,64 +1,33 @@
 <!doctype html>
 <html ng-app="liveblog-embed">
     <head>
-        <title>Embed</title>
+        <title>{{ blog.title }}</title>
```

Comment:
this is useless, we need to think on a plugin which will do some stuff for iframed window.